### PR TITLE
Quicktake: Fix textual visualizations look.

### LINKED ISSUE #933: Text visualization looks bad.
image

There is no spacing on the left side, the font looks bad (it should be of the same family as rest of the gui), the font is too small and too gray.

### PR DESCRIPTION
**Pull request by BinarySoftware**
### Pull Request Description
Fixes text-based visualizations so that they are consistent with IDE.
Closes #933 

Zrzut ekranu 2020-11-25 o 00 00 57

Zrzut ekranu 2020-11-25 o 00 10 31

### Important Notes
It cant affect IDE or introduce any regression. just a stylistic change. Nothing to worry about.

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/luna/enso/blob/main/docs/style-guide/rust.md) style guide.
- [x] All code has automatic tests where possible.
- [x] All code has been profiled where possible.
- [x] All code has been manually tested in the IDE.
- [x] All code has been manually tested in the "debug/interface" scene.
- [x] All code has been manually tested by the PR owner against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).
- [x] All code has been manually tested by at least one reviewer against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).

### CODE REVIEW

File: src/rust/ide/view/graph-editor/src/builtin/visualization/native/raw_text.rs [hunk 1]
```diff
@@ -119,6 +111,8 @@ impl RawTextModel {
     }
 
     fn set_size(&self, size:Vector2) {
+        let x_mod = size.x - 5.0;
```
Comment (wdanilo) [inline src/rust/ide/view/graph-editor/src/builtin/visualization/native/raw_text.rs:114]: hardcoded literal. Please move it to constant with a good name.

**Review by wdanilo — APPROVED** [review]
Looks good!

**Review by wdanilo — CHANGES_REQUESTED** [review]
Actually, one thing to be fixed visually - the padding. It should be equal to text padding on nodes. Right now, the raw text visualization has a big top padding and small left one. They should be equal and equal to the padding of text inside of nodes. The same applies to other visualizations. Please also consider right padding - please show the visualization with a long text to see how bottom and right padding behaves.

**Comment by BinarySoftware** [comment]
Better paddings:
Zrzut ekranu 2020-11-25 o 15 08 33

**Comment by BinarySoftware** [comment]
After talk with @farmaazon we've agreed that fixing scrollbars is bigger task, and it can be done in a separate PR. As we understand, @wdanilo, You want scrollbars to look like in safari, that their track doesnt overlap the background, which is non-trivial to actually implement correctly ;) Current implementation saves the day by disabling text to go out of bounds, especially that the visualization background has rounded corners, and that it is not a parent div to the visualization, rather that visualization "sits" on top of it.

**Comment by wdanilo** [comment]
paddings

@BinarySoftware are you sure these paddings are correct? Please make a screenshot with a node containing `{` as the first character and leats measure it on the screenshot

**Comment by BinarySoftware** [comment]
Ok, so nodes actually have different paddings themselves, while the text vis has equal ones. Actually by creating a one-letter node i feel that the text is slightly off to the right- on node that is.
Zrzut ekranu 2020-11-26 o 23 23 39

**Review by wdanilo — APPROVED** [review]

**Pull request merged**